api/v1/helpers/verbs/doPost.js

```diff
@@ -97,6 +96,10 @@ function doPost(req, res, next, props) {
   }
 
   return postPromise.then((o) => {
+    if (!o) { // could be here from u.handleError
+      return;
```

Comment:
I kind of think o will never be "falsey". None of the postPromise will return a "falsey" value. The post promise will either throw an error which will be caught by the catch block below or it will return a created instance. Correct?